### ISSUE TITLE
Failed to set preferences: Unable to write Firefox profile: The process cannot access the file because it is being used by another process.

### ISSUE THREAD

**Issue by Sigalov** [SeleniumHQ/selenium#6697]
Hello,
My setup is 10 Windows 8.1 VMs registered to another VM running hub.
Node example: java -Dwebdriver.gecko.driver=c:\selenium\geckodriver.exe -jar selenium-server-standalone-3.141.59.jar -role node -hub http://10.0.80.5:4444/grid/register -port 6558 -browser browserName=firefox,platform=WINDOWS,maxInstances=3,applicationName=qaKmsFrontEnd5 -maxSession 3 -host il-AutoKmsNode5-qa

Hub: java -jar ./selenium-server-standalone-3.141.59.jar  -role hub -host 10.0.80.5 -port 4444

Also added to windows path the gekodriver location and FF location.

This is from the code:
        fp = webdriver.FirefoxProfile()
        fp.set_preference('browser.download.folderList', 2) # custom location
        fp.set_preference('browser.download.manager.showWhenStarting', False)
        fp.set_preference('browser.download.dir', localSettings.LOCAL_SETTINGS_TEMP_DOWNLOADS)
        fp.set_preference('browser.helperApps.neverAsk.saveToDisk', 'video/mpeg,video/avi,video/MP2T,video/3gpp)
        fp.update_preferences()

webdriver.Remote(browser_profile=fp,command_executor=localSettings.LOCAL_SETTINGS_SELENIUM_HUB_URL, desired_capabilities={'marionette':True,'browserName': hostBrowser.split("_")[1], 'requireWindowFocus':True, 'applicationName': localSettings.LOCAL_SETTINGS_SELENIUM_GRID_POOL})

The code worked for me, suddenly I got an exception:
'Message: Failed to set preferences: Unable to write Firefox profile: The process cannot access the file because it is being used by another process. (os error 32)'

See full log: https://tinyurl.com/y7c22t7x

Maybe it caused by automatic update of FF.
I've updated everything to latest, and still getting the exception sometimes. It's not consistent, not in the same test and not the same VM.

Bottom line this is my current environment:
OS: Windows 8.1
Browser: FF 61.0.1 (Also tried on the latest 63) 64bit
Browser Driver version:  GeckoDriver 0.23
Python 3.5.2
Java Installed on Nodes (build 1.8.0_191-b12)
Selenium Grid version: selenium-server-standalone-3.141.59.jar

Some ideas?

**Comment by barancev** [comment]
As far as you run browser remotely we need a log of selenium server to diagnose the issue. If you can start selenium server with -debug option and collect the debug log it would be even better.

**Comment by barancev** [comment]
How often does this happen? I tried to run this sample of code in a loop 100 times (using selenium grid v.3.141.59, geckodriver 0.24 and firefox nightly), all passed.

**Comment by lmtierney** [comment]
Closing due to lack of response from issue creator
